### ISSUE TITLE
Interfacing with react.js

### ISSUE THREAD

**Issue by janat08** [peerlibrary/meteor-blaze-components#147]
Would it possible to try and interface with react.js, since the idea with OOP components is that they're shareable where react.js has more popularity.

**Comment by janat08** [comment]
this belongs in common package supposedly.

**Comment by janat08** [comment]
is that something that could even be well done, where you use react for boiler plate that would come with writing a widget avoiding react.js easily.

**Comment by mitar** [comment]
I do not understand what you have in mind here?

**Comment by janat08** [comment]
I suppose mapping api calls between each other such that you can override them and etc, so that importing react component doesn't mean that it's alien or that you have to write in react.js further down the chain. I hardly know react.js.

**Comment by janat08** [comment]
edited
```
class convertedWidget extends ReactConverter
mixin(){return [reactWidget]}
```

They say react.js constantly breaks it API and has no computed property or if for that matter. So basically you take it's render() method and create a template out of it that you then pass into convertedWidget template() method. With like 3 state management packages for react and it being able to compile to native mobile, it should be easy to recompile to blaze-components.

**Comment by mitar** [comment]
Yes. That was somehow planned with the whole idea of base component which would then be extended into react and blaze component. But then, somehow I never got to do it.
